bodhi/server/mail.py

```diff
@@ -325,8 +325,9 @@ def get_template(update, use_template='fedora_errata_template'):
                     if parent and not bug.parent:
                         log.debug("Skipping tracker bug %s" % bug)
                         continue
-                title = (
-                    bug.title != 'Unable to fetch title' and bug.title != 'Invalid bug number') \
+                title = (bug.title != 'Unable to fetch title'
+                    and bug.title != 'Invalid bug number'
+                    and bug.title != 'Private bug') \
```

Comment:
```suggestion
                    and not bug.private) \
```

Also, ```flake8``` is not happy with these lines:

```
04:25:11 ./bodhi/server/mail.py:329:21: W503 line break before binary operator
04:25:11                     and bug.title != 'Invalid bug number'
04:25:11                     ^
04:25:11 ./bodhi/server/mail.py:329:21: E128 continuation line under-indented for visual indent
04:25:11                     and bug.title != 'Invalid bug number'
04:25:11                     ^
04:25:11 ./bodhi/server/mail.py:330:21: W503 line break before binary operator
04:25:11                     and bug.title != 'Private bug') \
04:25:11                     ^
04:25:11 ./bodhi/server/mail.py:330:21: E128 continuation line under-indented for visual indent
04:25:11                     and bug.title != 'Private bug') \
```